tabs/src/components/SubTaskBox/SubTaskBox.js

```diff
@@ -0,0 +1,54 @@
+import Styled from './SubTaskBox.styles';
+
+import { Checkbox } from '@fluentui/react';
+import { people } from '@fluentui/example-data';
+import { Persona, PersonaPresence } from '@fluentui/react/lib/Persona';
+import { TestImages } from '@fluentui/example-data';
+
+const PersonaList = (people) => {
+  const peopleLen = Object.values(people).length;
+  let extraNum = peopleLen > 5 ? peopleLen - 5 : 0;
```

Comment:
```suggestion
  const extraNum = peopleLen > 5 ? peopleLen - 5 : 0;
```

다시 할당되는 경우가 없으니 const를 적용하면 좋겠네요